### PR TITLE
Initial Getting Started ideas

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
Here is my initial ideas for a getting started overhaul. I couldn't quite figure out your Jekyll setup to get this to render or appear in menus, might need some guidance on that.

I created a top level "getting started" guide that covers an introduction, common install options, basic terminology, and then some first commands.

Potential todos / next steps:

- Add an intro to why use Geth (might be covered elsewhere)
- Potentially add some more details in a few places
- Continue polishing the various links linked to from this page. Started some.
- Think about next steps for commands / using Geth, and add a link
- I have some style and terminology questions (i.e. what word to use for certain terms as there's inconsistency), but that can wait.

And of course any feedback you all may have.

### CODE REVIEW

File: docs/_install-and-build/Installation-Instructions-for-Mac.md [hunk 1]
```diff
@@ -25,7 +26,7 @@ You should now be able to run `geth` and connect to the network.
 
 Make sure to check the different options and commands with `geth --help`
 
-For options and patches, see: https://github.com/ethereum/homebrew-ethereum
+For options and patches, see: <https://github.com/ethereum/homebrew-ethereum>
```
Comment (adamschmideg) [inline docs/_install-and-build/Installation-Instructions-for-Mac.md]: Why do you prefer urls in angle brackets? I don't mind either way, just asking.

Comment (ChrisChinchilla) [reply docs/_install-and-build/Installation-Instructions-for-Mac.md]: @adamschmideg This is done automatically by my markdown linter, for what markdown calls "naked" URLs, i.e. those without link text, this is the correct syntax, and turns them into clickable links, whereas, I think the former isn't.

**Comment by ChrisChinchilla** [comment]
@adamschmideg I still need to figure out the proper build setup for your Jekyll installation too, might quickly try that again now. As this is still fairly short, I could also add your suggestions here, if it gets too long with those examples, maybe split into a next step, I'll try it first.

**Comment by holiman** [comment]
@ChrisChinchilla I used this guide to setup jekylll locally and have it rendered like GH does: https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll

**Comment by ChrisChinchilla** [comment]
@holiman I know Jekyll very well, what I'm trying to figure out is your particular config, which is different from what I've seen before ;)

Think I just figured it out.

**Pull request merged**